internal/backend/temporalclient/client.go

```diff
@@ -142,28 +150,22 @@ func (c *impl) healthcheck(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-	c.z.Debug("temporal reports healthy")
+	c.z.Debug("Connection to Temporal is healthy")
 
 	return nil
 }
 
 func (c *impl) Start(context.Context) error {
 	if c.cfg.Monitor.CheckHealthInterval == 0 {
-		c.z.Warn("periodical check health is disabled")
+		c.z.Warn("Periodic Temporal health checks are disabled")
 		return nil
 	}
 
-	var ok bool
-
 	go func() {
 		for {
-			if err := c.healthcheck(context.Background()); err != nil {
+			if err := c.healthCheck(context.Background()); err != nil {
 				// TODO: stats.
-				ok = false
-				c.z.Error("temporal check health error", zap.Error(err))
-			} else if !ok {
-				ok = true
-				c.z.Info("temporal reports healthy")
+				c.z.Error("Temporal health check error", zap.Error(err))
```

Comment:
IMO, in terms of logs it's nice to have what u've removed - the single report that connection is back.

Reply:
I get your point, added it back.